examples/aishell3/vc1/README.md

```diff
@@ -38,6 +38,10 @@ Run the command below to
 ```bash
 ./run.sh
 ```
+You can choose the stage range you want to run, for example, run the following command will only preprocess the dataset.
```

Comment:
You can choose a range of stages you want to run, or set ```stage``` equal to ```stop-stage``` to only use one stage, for example, run the following command will only preprocess the dataset.

Reply:
to use only one stage